src/mongo/registerMongoCommands.ts

```diff
@@ -93,6 +93,15 @@ export function registerMongoCommands(editorManager: CosmosEditorManager): void
             languageClient.disconnect();
         }
     });
+    registerCommand('cosmosDB.disconnectMongoDB', async (context: IActionContext, node?: MongoDatabaseTreeItem) => {
```

Comment:
I think it's a little confusing to have a "disconnect" and "detach". I'd prefer if this was just part of "detach"